examples/simple-examples/src/numbers/available/rentAny.ts

```diff
@@ -17,6 +18,7 @@ import {
 
   const servicePlanId = readServicePlanId();
   const appId = readApplicationKey();
+  const callbackUrl = getNumberCallbackUrlFromConfig();
 
   if (!servicePlanId && !appId) {
```

Comment:
Does this test is not managed by client itself ?

Reply:
No, here the servicePlanId and the appId values are read from the configuration to assign a number to a SMS app or a Voice app.
As we are in the number app, the sinch client is using the projectId credentials and doesn't know about the SMS app(s) and Voice app(s)

Reply:
Ok, thank you, I missed this point